Dockerfile.train

```diff
@@ -1,39 +1,67 @@
-# Please refer to the TRAINING documentation, "Basic Dockerfile for training"
+# This is a Dockerfile useful for training models with Coqui STT.
+# You can train "acoustic" models with audio + Tensorflow, and
+# you can train "language" models with text + KenLM.
+
+FROM ubuntu:20.04 AS kenlm-build
+ENV DEBIAN_FRONTEND=noninteractive
+
+RUN apt-get update && \
+    apt-get install -y --no-install-recommends \
+    build-essential cmake libboost-system-dev \
+    libboost-thread-dev libboost-program-options-dev \
+    libboost-test-dev libeigen3-dev zlib1g-dev \
+    libbz2-dev liblzma-dev
+
+# Build KenLM to generate new scorers
+WORKDIR /code
+COPY kenlm /code/kenlm
+RUN cd /code/kenlm && \
+    mkdir -p build && \
+    cd build && \
+    cmake .. && \
+    make -j $(nproc) || \
+    ( echo "ERROR: Failed to build KenLM."; \
+    echo "ERROR: You must update the kenlm submodule on host before building this Dockerfile."; \
```

Comment:
This error message is too assertive when this is only one of the reasons the build could fail. Maybe replace "You must" with "Make sure you".

Reply:
Agree